frontend/lib/ui/dashboard/testresults_components.dart

```diff
@@ -0,0 +1,106 @@
+import 'package:flutter/material.dart';
+import 'package:yaru_widgets/widgets.dart';
+
+import '../../models/test_results.dart';
+import '../spacing.dart';
+
+class GroupTestResultView extends StatelessWidget {
+  const GroupTestResultView({
+    super.key,
+    required this.testResultStatus,
+    required this.groupTestResults,
+  });
+
+  final List<TestResult> groupTestResults;
+  final TestResultStatus testResultStatus;
+
+  @override
+  Widget build(BuildContext context) {
+    return YaruExpandable(
+      header: Row(
+        children: [
+          testResultStatus.icon,
+          const SizedBox(width: Spacing.level4),
+          Text(
+            testResultStatus.name,
+            style: Theme.of(context).textTheme.titleLarge,
+          ),
+          const Spacer(),
+          Text(
+            groupTestResults.length.toString(),
+            style: Theme.of(context).textTheme.titleLarge,
+          ),
```

Comment:
I'm not sure if there were any discussions on this already, but it seems more clear to me to have the number of test results on the left not right. But this is open for discussion